amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/styles.scss

```diff
@@ -82,4 +86,60 @@
       margin-top: 4px;
     }
   }
+
+  &:hover {
+    .more-icon {
+      visibility: visible;
+    }
+  }
+
+  .open {
+    .more-icon {
+      box-shadow: none;
+      visibility: visible;
+    }
+  }
+
+  .more-icon {
+    border-style: none;
+    visibility: hidden;
+    border-radius: 50%;
```

Comment:
Based on this mock, I'm not sure this is supposed to be a circle.

<img width="291" alt="Screen Shot 2019-08-15 at 11 56 20 AM" src="https://user-images.githubusercontent.com/4338702/63118916-b87e8180-bf53-11e9-9d1f-3d7eb67b1a2b.png">